### PR TITLE
Update grpc-netty & Remove jetcd-core

### PR DESCRIPTION
**Pull request by danilbykov**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -47,29 +47,17 @@ lazy val etcd = project
...
       "com.thesamet.scalapb" %% "scalapb-runtime-grpc"           % scalapb.compiler.Version.scalapbVersion,
-      "com.coreos"           % "jetcd-core"                      % "0.0.2" excludeAll (ExclusionRule(organization = "io.grpc")),
-      "com.coreos"           % "jetcd-common"                    % "0.0.2" excludeAll (ExclusionRule(organization = "io.grpc")),
       "com.pauldijou"        %% "jwt-core"                       % "4.2.0"
+    ),
+    Compile / PB.targets := Seq(
+      scalapb.gen() -> (Compile / sourceManaged).value / "scalapb"
     )
   )
-  .settings {
```
Comment (fit51) [inline build.sbt:60]:
```
.settings {
    val generateSources = TaskKey.apply[Unit]("generateSources")

    def genPackage(f: File): File = f / "com" / "github" / "fit51" / "reactiveconfig" / "etcd" / "gen"
    generateSources := {
      Compile / PB.targets := Seq(
        scalapb.gen() -> (Compile / sourceManaged).value / "scalapb"
      )
      (PB.generate in Compile).value
      val genDir    = genPackage((sourceManaged in Compile).value / "scalapb")
      val targetDir = genPackage((sourceDirectory in Compile).value / "scala")
      println(s"Generated in: $genDir")
      println(s"Moved to: $targetDir")
      IO.copyDirectory(genDir, targetDir, true, true)
      IO.delete(genDir)
    }
  }
```
Tested it, seems to work fine

File: etcd/src/main/scala/com/github/fit51/reactiveconfig/etcd/ChannelManager.scala [hunk 2]
```diff
@@ -41,12 +41,41 @@ object ChannelManager {
...
+      .map(e => e.getHost() + (if (e.getPort() != -1) s":${e.getPort()}" else ""))
+      .mkString(",")
+    val target = s"https://${authority.getOrElse("etcd")}/$uris"
+    new ChannelManager(target, options, authority, trustManagerFactory) with Authorization {
       override val credentials    = credential
       override implicit val clock = cl
     }
   }
+
+  NameResolverRegistry
```
Comment (fit51) [inline etcd/src/main/scala/com/github/fit51/reactiveconfig/etcd/ChannelManager.scala:56]: I wonder, if this Resolver would be used in netty-code all over the project?
Wouldn't it affect other netty integrations?

Comment (fit51) [reply etcd/src/main/scala/com/github/fit51/reactiveconfig/etcd/ChannelManager.scala:56]: In that case, mb we should create a custom scheme (ex. etcd://) to isolate our custom registry?

Comment (fit51) [reply etcd/src/main/scala/com/github/fit51/reactiveconfig/etcd/ChannelManager.scala:56]: Have a look [here](https://github.com/grpc/grpc-java/issues/7133), guys are using the following kind of scheme `etcd:///authority/host1:port,host2:port`

**Review by fit51 — CHANGES_REQUESTED** [review]

File: build.sbt (hunk 1 above)
Comment (danilbykov) [reply build.sbt:60]: done

File: etcd/src/main/scala/com/github/fit51/reactiveconfig/etcd/ChannelManager.scala (hunk 2 above)
Comment (danilbykov) [reply etcd/src/main/scala/com/github/fit51/reactiveconfig/etcd/ChannelManager.scala:56]: done

**Review by fit51 — APPROVED** [review]

**Pull request merged**